packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping.md

```diff
@@ -102,30 +104,14 @@ _Props/Attrs_
 
 -   `field-name`: string
 -   `key-type`: 'string' | 'number' | 'boolean'
+-   `display`: enum with possible values of `iconOnly`, `labelOnly`, and `iconAndLabel` - Specifies what is rendered in a cell for the column. Note, that this attribute only affects a cell's rendering; a group row will always render both the icon and the label.
```

Comment:
I went back and forth on this attribute. I considered making it a boolean (`label-hidden`) to align with a number of our other components (e.g. `content-hidden` on buttons, `underline-hidden` on anchors, `title-hidden` on banners & dialogs, etc).

However, I decided that making it an enum was probably a better decision because:
1. We could do it in a backwards compatible way if we make the default value `iconOnly`
2. It allows us to add more configurations in the future without making a breaking change to the API.

Reply:
This is now on the mapping element itself (specifically for icon and spinner mappings), and it is a boolean called `label-hidden`.